plugins/inputs/opcua_listener/integration_test.go

```diff
@@ -0,0 +1,140 @@
+package opcua_listener
```

Comment:
Can you rename this to `opcua_listener_test.go` to be consistent with other plugins. Prefer to have tests related to a specific file to be file_test.go. Thanks

Reply:
Merged with `opcua_listener_test.go`